src/sonic-config-engine/sonic-cfggen

```diff
@@ -237,11 +244,11 @@ def main():
         minigraph = args.minigraph
         if platform:
             if args.port_config != None:
-                deep_update(data, parse_xml(minigraph, platform, args.port_config))
+                deep_update(data, parse_xml(minigraph, platform, args.port_config,hostname=asic_name))
             else:
-                deep_update(data, parse_xml(minigraph, platform))
+                deep_update(data, parse_xml(minigraph, platform,hostname=asic_name))
         else:
-            deep_update(data, parse_xml(minigraph, port_config_file=args.port_config))
+            deep_update(data, parse_xml(minigraph, port_config_file=args.port_config,hostname=asic_name))
```

Comment:
can you correct the coding style? need extra space after the ','

Reply:
fixed spacing.